webapp/app/routes/app/transactions.$hash/_route.tsx

```diff
@@ -12,14 +12,21 @@ import {
 import { displayBytes32 } from "@/routes/app/proposals/$id/common-tables";
 import { getTransactionUrl } from "@/utils/etherscan";
 import { capitalizeFirstLetter } from "@/utils/string";
-import { useNavigate, useParams } from "@remix-run/react";
+import { env } from "@config/env.server";
+import { json } from "@remix-run/node";
+import { useLoaderData, useNavigate, useParams } from "@remix-run/react";
 import { ArrowLeft, CircleCheckBig, SquareArrowOutUpRight } from "lucide-react";
 import { type Hex, formatEther, formatGwei } from "viem";
 import { useWaitForTransactionReceipt } from "wagmi";
 
+export async function loader() {
+  return json({ ethNetwork: env.ETH_NETWORK });
+}
+
```

Comment:
The loader could return the transaction url directly to the page. In case the network is necessary anyway, the same pattern can be used as in https://github.com/ZKsync-Association/zksync-upgrade-verification-tool/blob/ed52a2ae8e8b89be77de6346725292586ce32bb7/webapp/app/components/env-badge.tsx